controllers/solrcloud_controller.go

```diff
@@ -716,7 +768,9 @@ func (r *SolrCloudReconciler) SetupWithManagerAndReconciler(mgr ctrl.Manager, re
 		Owns(&corev1.ConfigMap{}).
 		Owns(&appsv1.StatefulSet{}).
 		Owns(&corev1.Service{}).
-		Owns(&extv1.Ingress{})
+		Owns(&extv1.Ingress{}).
+		Owns(&corev1.Secret{}).
```

Comment:
We just watch secrets, not create them I think. This will require that lovely `indexAndWatchForTlsSecrets()` method. I can handle this though!